intelmq/lib/upgrades.py

```diff
@@ -532,6 +534,41 @@ def v230_csv_parser_parameter_fix_1(defaults, runtime, harmonization, dry_run):
     return changed, defaults, runtime, harmonization
 
 
+def v300_deprecations(defaults, runtime, harmonization, dry_run):
+    """
+    Deprecate malwaredomainlist parser
+    """
+    changed = None
+    malwaredomainlistparser = []
+    for bot_id, bot in runtime.items():
+        if bot["module"] == "intelmq.bots.parsers.malwaredomainlist.parser":
+            malwaredomainlistparser.append(bot_id)
+    for bot_id in malwaredomainlistparser:
+        del runtime[bot_id]
```

Comment:
That could cause troubles as the pipeline and (to a lesser degree) the positions need to be adapted as well.

Reply:
Fixed, it doesn't change anything, but emits a warning